app/core/Tracker.coffee

```diff
@@ -154,6 +154,7 @@ module.exports = class Tracker extends CocoClass
       'Simulator Result',
       'Started Level Load', 'Finished Level Load',
       'Start HoC Campaign', 'Show Amazon Modal Button', 'Click Amazon Modal Button', 'Click Amazon link',
+      'Error in ssoConfirmView'
```

Comment:
This is intended to be temporary right?  Lets add a comment or TODO to remind ourselves why this is here / to remove it later so we don't come back in a few months and wonder what this is for.

Reply:
Yeah, that's a good point. Will do.